package.json

```diff
@@ -131,8 +132,15 @@
   "scripts": {
     "lint": "eslint \"**/*.ts\" --fix",
     "lint:ci": "eslint \"**/*.ts\" --max-warnings 0",
-    "dev": "rm -rf lib && tsc -b && ./bin/run",
-    "prepack": "rm -rf lib && tsc -b",
+    "pack-hdf-converters": "run-script-os",
+    "pack-hdf-converters:win32": "pack-hdf-converters.bat",
+    "pack-hdf-converters:darwin:linux": "./pack-hdf-converters.sh",
+    "dev": "run-script-os",
+    "dev:darwin:linux": "rm -rf lib && tsc -b && ./bin/run",
+    "dev:win32": "(IF EXIST lib rmdir /s /q lib) && tsc -b && .\bin\run",
```

Comment:
consistently put windows on top cause i think that's what you're doing in heimdall

Reply:
Done!